diff --git a/meta-arago-extras/recipes-connectivity/bluez-tools/bluez-tools_git.bb b/meta-arago-extras/recipes-connectivity/bluez-tools/bluez-tools_git.bb
index 5ef5046..9cef87e 100644
--- a/meta-arago-extras/recipes-connectivity/bluez-tools/bluez-tools_git.bb
+++ b/meta-arago-extras/recipes-connectivity/bluez-tools/bluez-tools_git.bb
@@ -12,7 +12,7 @@ SRCREV = "171181b6ef6c94aefc828dc7fd8de136b9f97532"
 
 S = "${WORKDIR}/git"
 
-SRC_URI = "git://gitorious.org/bluez-tools/bluez-tools.git;protocol=git"
+SRC_URI = "git://git.baserock.org/delta/bluez-tools.git;protocol=git"
 
 inherit autotools
 
